src/App.jsx

```diff
@@ -7,24 +7,29 @@ import EventDetailsPage from "./pages/EventDetailsPage";
 import CreateEvent from "./pages/CreateEvent";
 import TicketsPage from "./pages/TicketsPage";
 import CommunityPage from "./pages/CommunityPage";
+import Participants from "./pages/Participants";
 
 function App() {
-  return (
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<HomePage />} />
-        <Route path="create-event" element={<CreateEvent />} />
-        <Route path="app" element={<AppLayout />}>
-          <Route index element={<Navigate replace to="home" />} />
-          <Route path="home" element={<DashboardPage />} />
-          <Route path="events" element={<EventsPage />} />
-          <Route path="event/:eventId" element={<EventDetailsPage />} />
-          <Route path="tickets" element={<TicketsPage />} />
-          <Route path="community" element={<CommunityPage />} />
-        </Route>
-      </Routes>
-    </BrowserRouter>
-  );
+    return (
+        <BrowserRouter>
+            <Routes>
+                <Route path="/" element={<HomePage />} />
+                <Route path="create-event" element={<CreateEvent />} />
+                <Route path="app" element={<AppLayout />}>
+                    <Route index element={<Navigate replace to="home" />} />
+                    <Route path="home" element={<DashboardPage />} />
+                    <Route path="events" element={<EventsPage />} />
+                    <Route
+                        path="event/:eventId"
+                        element={<EventDetailsPage />}
+                    />
+                    <Route path="tickets" element={<TicketsPage />} />
+                    <Route path="community" element={<CommunityPage />} />
+                    <Route path="participants" element={<Participants />} />
```

Comment:
The Route to this page is not linking 

"/participants"
keep throwing  blank page